src/app/blog/qad/page.mdx

```diff
@@ -0,0 +1,209 @@
+import { BlogPostLayout } from '@/components/BlogPostLayout'
+import {ThemeImage} from '@/components/ThemeImage'
+
+export const post = {
+  draft: false,
+  author: 'Floris Bruynooghe',
+  date: '2025-08-26',
+  title: 'QUIC Address Discovery',
```

Comment:
Let's make this title more fun.
```suggestion
  title: 'Ditching STUN: QUIC Address Discovery',
```

```suggestion
  title: 'Towards standardized QUIC hole punching: QUIC Address Discovery',
```

Reply:
- Ditching STUN for QUIC Address Discovery
- Moving from STUN to QUIC Address Discovery

I appreciate I'm just making it more descriptive rather than fun.  But that probably makes sense when you think of it showing up on sites like reddit, lobsters, hn, ...

Reply:
It's already renamed to "Moving from STUN to ..." :)
And I think that name is good fwiw